third_party/bigframes_vendored/pandas/core/config_init.py

```diff
@@ -26,25 +27,32 @@
     Computation deferred. Computation will process 28.9 kB
 
 Users can also get a dry run of the job by accessing the query_job property before they've run the job. This will return a dry run instance of the job they can inspect.
+
     >>> df.query_job.total_bytes_processed
     28947
 
 User can execute the job by calling .to_pandas()
+
     >>> # df.to_pandas()
```

Comment:
should we remove `# ` here?

Reply:
I think let's keep it, my guess is that it is there to save downloading data to let the doctest run faster